gluonnlp/data/utils.py

```diff
@@ -307,6 +307,14 @@ def _get_home_dir():
     return _home_dir
 
 
+def _get_stanford_home_dir():
+    """Get home directory for stanford toolkits"""
+    _stanford_home_dir = os.environ.get('STANFORD_HOME',
+                                        os.path.join(_get_home_dir(), 'stanford'))
```

Comment:
don't define a new environment variable or otherwise it's hard for people to discover.
use os.path.join(_get_home_dir(), 'stanford-segmenter') instead.

Reply:
OK. I will update the constructor of NLTKStanfordSegmenter. 
Also, I do not remove the function `_get_stanford_home_dir` since it may be used in the future.

Reply:
Please remove dead code. It is one source of technical debts.